rio/src/screen/mouse.rs

```diff
@@ -53,6 +54,18 @@ impl Default for Mouse {
     }
 }
 
+impl Mouse {
+    pub fn new(config: &Rc<rio_config::Config>) -> Self {
+        Self {
+            multiplier: config.scroll_multiplier,
+            ..Default::default()
+        }
+    }
+    pub fn update_multiplier(&mut self, config: &Rc<rio_config::Config>) {
```

Comment:
please rename to set_multiplier to keep consistency with other parts of rio, like https://github.com/raphamorim/rio/blob/d6b73d4a2fc2cfcf91160dfc883c8f48174fe3d4/rio/src/ime.rs#L17 , https://github.com/raphamorim/rio/blob/d6b73d4a2fc2cfcf91160dfc883c8f48174fe3d4/sugarloaf/src/font/loader/mod.rs#L420 , https://github.com/raphamorim/rio/blob/d6b73d4a2fc2cfcf91160dfc883c8f48174fe3d4/sugarloaf/src/core.rs#L43

Reply:
changed the name. f9d755f41fbfa3d16fdfbd7a25506f0b099b7cfb. should we call it in Screen::update_config?

Reply:
I think `update_config` should actually rename to other to something else tbh. The prefix `set_` is the default for get/set in rio but update_config does more than a regular setter, so idk (in the future I plan to revisit update_config though)